src/lamp_py/ingestion/ingest.py

```diff
@@ -26,18 +26,6 @@ def convert(self) -> None:
         )
 
 
-def get_converter(
-    config_type: ConfigType, metadata_queue: Queue[Optional[str]]
-) -> Converter:
-    """
-    get the correct converter for this config type. it may raise an exception if
-    the gtfs_rt file type does not have an implemented detail
-    """
-    if config_type.is_gtfs():
-        return GtfsConverter(config_type, metadata_queue)
-    return GtfsRtConverter(config_type, metadata_queue)
-
-
```

Comment:
don't need this anymore since GtfsConverter class operates on a stand-alone basis